configs/textdet/dbnet/README.md

```diff
@@ -16,24 +16,24 @@ Recently, segmentation-based methods are quite popular in scene text detection,
 
 ### SynthText
 
-|                                  Method                                   | Backbone | Training set | #iters  |                                               Download                                               |
-| :-----------------------------------------------------------------------: | :------: | :----------: | :-----: | :--------------------------------------------------------------------------------------------------: |
-| [DBNet_r18](/configs/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext.py) | ResNet18 |  SynthText   | 100,000 | [model](https://download.openmmlab.com/mmocr/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext/dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth) \| [log](https://download.openmmlab.com/mmocr/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext/20221214_150351.log) |
+|                                  Method                                   | Backbone | Training set | #iters  |                                          Download                                          | Batch Size |
+| :-----------------------------------------------------------------------: | :------: | :----------: | :-----: | :----------------------------------------------------------------------------------------: | :--------: |
+| [DBNet_r18](/configs/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext.py) | ResNet18 |  SynthText   | 100,000 | [model](https://download.openmmlab.com/mmocr/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext/dbnet_resnet18_fpnc_100k_synthtext-2e9bf392.pth) \| [log](https://download.openmmlab.com/mmocr/textdet/dbnet/dbnet_resnet18_fpnc_100k_synthtext/20221214_150351.log) |   1xb16    |
```

Comment:
Need to change the batch size format to avoid confusion for beginners:
`1xb16` -> `1 x 16`.

Same for others.